infer_structcol/tests/test_model.py

```diff
@@ -9,11 +9,11 @@
 from pandas.util.testing import assert_frame_equal
 
 def test_calc_model_spect():
-    sample = Sample(500, 100, 200, 1.5, 1)
-    theta = (0.5, 0, 0, 0, 0)
+    sample = Sample(500, 1.5, 1)
+    theta = (0.5, 100, 200, 0, 0, 0, 0)
     assert_frame_equal(calc_model_spect(sample, theta, 2), Spectrum(500, 
-                       reflectance = 0.813006364656, sigma_r = 0.025358376581, 
```

Comment:
Why have these values changed? I think we are just moving radius and thickness information from sample into theta, so we wouldn't expect the output to change. If that did change the returned spectrum, it should be treated as a bug and we need to track down why that changed things.